subscription_oca/models/sale_order_line.py

```diff
@@ -14,4 +14,9 @@ def get_subscription_line_values(self):
             "price_unit": self.price_unit,
             "discount": self.discount,
             "price_subtotal": self.price_subtotal,
+            "analytic_distribution": self.analytic_distribution
+            if self.analytic_distribution
+            else {self.project_id.analytic_account_id.id: 100}
```

Comment:
Does this need project to be installed?

Reply:
@eLBati Agree with @tarteo here.

Reply:
Right, thanks, we are going to put the changes in a dedicated module

Reply:
Fixed this and created https://github.com/OCA/contract/pull/1270